### ISSUE TITLE
Decimal places

### ISSUE THREAD

**Issue by ddaddy** [hankinsoft/AppWage#10]
I'd also like to say thank you very much for this app. I certainly would have paid for an app like this.

A slight bug I found:
See the attached image. How can number of downloads include 0.4 of a download and also Refunds includes 0.5535 of a refund.

<img width="368" alt="screen shot 2016-11-08 at 11 01 26" src="https://cloud.githubusercontent.com/assets/1425808/20096665/dcaf22c6-a5a2-11e6-80f2-376c377bea2b.png">

**Comment by hankinsoft** [comment]
tldr; this is a side-effect of how app wage calculates daily sales.

Longer details:
AppWage tries to have a 'daily' value for every day. Apple only provides true daily sales for the previous week (maybe two, I can't remember straight off). They also provide weekly, monthly and yearly data as well, which AppWage will attempt to convert into daily sales. An example which would show this issue would be as follows:

You only have two reports from apple.
November 2'nd you have a daily report with 1 refund.
You have a weekly report with 9 refunds. Your weekly report includes November 2nd.

This means during the week we had 9 total refunds. We know that November 2nd had one refund, but we do not know which days had the others. AppWage will take the remaining six days of the week and average the total refunds, ending up being 1.3333 (8 refunds in 6 days). So now our week looks something like this:

Nov1 - 1.333 refunds
Nov2 - 1 refund
Nov3 - 1.333 refunds
... etc

Then for the total charts, we would sum the days (6 x 1.333 + 1) = 8.998. This unfortunately can lead to some slightly off totals (but not extremely far off). Hope that helps explain!

**Comment by ddaddy** [comment]
Great explanation. Thanks.
I didn't realise you/it went to such great lengths 👍 
I have all my daily reports from the day I started 4 years ago and imported them all into this app. I didn't think that you'd actually use the monthly/yearly reports to work out rough figures.

Totally unrelated, does the app keep hold of the reports somewhere of delete them after processing?

**Comment by hankinsoft** [comment]
Hi,
Yes downloaded reports are kept in:
~/Library/Application Support/com.hankinsoft.osx.appwage.iTunesCollectionService/iTunesReports

**Comment by ddaddy** [comment]
👍

**Comment by ddaddy** [comment]
I just noticed that imported reports are not saved. Is that a design choice or bug?

**Comment by hankinsoft** [comment]
This is by design.
